sentences/pt-br/fan_HassTurnOn.yaml

```diff
@@ -2,7 +2,16 @@ language: pt-br
 intents:
   HassTurnOn:
     data:
-      - sentences: []
+      - sentences:
+          - "<ligar> [todo[s]] [o[s]] ventilador[es] <na_zona>"
         slots:
-          domain: fan
-          name: all
+          domain: "fan"
+          name: "all"
+        response: fans_area
+      - sentences:
+          - "<ligar> (todo[s] [o[s]] ventilador[es] [da casa] | [todo[s]] [o[s]] ventilador[es] da casa)"
```

Comment:
Same as for turning off all the fans:
---
I think it makes no sense `todo[s]` and the two options inside `(<OPTION_1> | <OPTION_2>)` are basically the same, aren't they?

And since it's an action to turn all the fans on, `[os]` makes more sense than `[o[s]]`

What do you think of updating it to:
```
- sentences:
    - "<ligar> [todos] [os] ventiladores [da casa]"
```